board/xtsc/vp6-mp/Makefile.attr

```diff
@@ -0,0 +1,12 @@
+# Customer ID=4313; Build=0xa64b0; Copyright (c) 2023 by Cadence Design Systems.  ALL RIGHTS RESERVED.
+# These coded instructions, statements, and computer programs are the
+# copyrighted works and confidential proprietary information of Tensilica Inc.
+# They may not be modified, copied, reproduced, distributed, or disclosed to
+# third parties in any manner, medium, or form, in whole or in part, without
+# the prior written consent of Tensilica Inc.
```

Comment:
Everything else here is either MIT or dual MIT/GPL licensed. Are you sure you mean what's written here?

Reply:
Thank you for reviewing. I did not mean it; it is a copy and paste.
Removing those lines shortly.

Reply:
c134fd0c54cfde603532c85fa68fc3627dee8671